randomthought/views.py

```diff
@@ -0,0 +1,20 @@
+from django.shortcuts import render
+import random
+
+
+
+THOUGHT_LIST = [
+    'baba z wozu, koniom lżej',
+    'gdzie diabeł nie może, tam babę pośle',
+    'jeszcze się taki nie narodził, co by babie we wszystkim dogodził',
+]
+
+def ThoughtView(request):
```

Comment:
Konwencja nazewnicza w pythonie mówi, że nazwy funkcji (czyli też widoków django) powinny być w konwencji snake case (a nie kamel case), czyli tutaj powinno być thought_view (https://peps.python.org/pep-0008/#function-and-variable-names)